docs/en/enterprise-edition/policy-reference/azure-policies/azure-general-policies/bc-azure-244.adoc

```diff
@@ -0,0 +1,49 @@
+
+== Local users used for Azure Storage
+
+=== Policy Details
+
+[width=45%]
+[cols="1,1"]
+|===
+|Prisma Cloud Policy ID
+| TBD
+
+|Checkov ID
+| https://github.com/bridgecrewio/checkov/blob/main/checkov/terraform/checks/resource/azure/StorageLocalUsers.py[CKV_AZURE_244]
+
+|Severity
+|LOW
+
+|Subtype
+|Build
+
+|Frameworks
+|Terraform,TerraformPlan
+
+|===
+
+=== Description
+
+This policy is checking for the use of local users within Azure Storage accounts. Local users can pose a security risk because they might have direct access to the storage account, which can potentially lead to unauthorized data access or compromise if those credentials are exposed or not managed securely. The best practice is to use Azure Active Directory (AD) identities or managed identities to handle access control. This is preferred because Azure AD provides advanced security features, centralized management, and stronger authentication methods, reducing the risk of unauthorized access compared to local users that rely solely on static credentials.
+
+=== Fix - Buildtime
+
+*Terraform*
+
+* *Resource:* azurerm_storage_account
+* *Arguments:* local_user_enabled
+
+Avoid the use of local users for Azure Storage to enhance security and manageability. Instead, use Azure Active Directory or other centralized identity management solutions to control access to storage accounts. This reduces the dependency on local users, which can be less secure and harder to manage.
```

Comment:
Use Azure Active Directory or other centralized identity management solutions to control access to Azure Storage accounts. Relying on Azure AD reduces the dependency on static credentials. Avoid using local users, as they depend on static credentials, which can be less secure and harder to manage.